ompi/runtime/ompi_rte.c

```diff
@@ -69,6 +69,7 @@ opal_process_name_t pmix_name_invalid = {UINT32_MAX, UINT32_MAX};
  * session directory structure, then we shall cleanup after ourselves.
  */
 static bool destroy_job_session_dir = false;
+static bool destroy_proc_session_dir = false;
```

Comment:
Does `destroy_proc_session_dir` ever get set to `true`?

Reply:
Yeah, I missed it - added it now